modules/react-native-status/desktop/rctstatus.cpp

```diff
@@ -20,7 +20,7 @@
 #include <QVariantMap>
 #include <QDir>
 #include <QStandardPaths>
-#include <QtConcurrent>
+#include <future>
```

Comment:
Can you explain this change? Was nix unable to install `QtConcurrent`, or is this just some side change?

Reply:
Yeah, Nix's `qt5.full` package doesn't seem to include `QtConcurrent`, but even with that aside, I think it's not ideal to add a dependency when the standard library already provides the same functionality. I.e. even if this PR doesn't go forward, I'd still want to include this change in another targeted PR.

Reply:
QtConcurrent is part of `qt5.qtbase` which is included in `qt5.full`

Reply:
That was my understanding as well.

Reply:
What made me look into an alternative was this error right after I brought in the `qt5.full` Nix package:
```
CMake Error at /nix/store/d28n5vfavdlg0gd43cgd52hcyhm96xaz-qt-full-5.12.0/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:372 (message):
  Can not use "Concurrent" module which has not yet been found.
Call Stack (most recent call first):
  /home/pedro/src/github.com/status-im/status-react/node_modules/react-native/ReactQt/runtime/src/CMakeLists.txt:259 (qt5_use_modules)
```
I could take some time and delve into it, but the truth is that the reason we import QConcurrent is not strong enough when there is an equivalent solution with `std::async`.

Reply:
Huh, turned out to be a missing import in one file (no idea why this wasn't a problem pre-Nix). I'll revert the removal of QtConcurrent for now, to make the PR as simple as possible.